apps/container/src/views/HistoryView.vue

```diff
@@ -246,19 +266,48 @@
 }
 </style>
 <script setup lang="ts">
+import { useQuery } from '@tanstack/vue-query';
+import { conceptsColor } from 'consts';
+import { Enrollment, Enrollments, Users } from 'services';
+import { Concept } from 'types';
+import { computed, ref } from 'vue';
+import ReviewDialog from '@/components/ReviewDialog.vue';
+import CenteredLoading from '@/components/CenteredLoading.vue';
 import PaperCard from '@/components/PaperCard.vue';
 import TableComponent from '@/components/TableComponent.vue';
-import {
-  Users,
-  Enrollments,
-  Enrollment as EnrollmentType,
-  Concept,
-} from 'services';
-import { computed } from 'vue';
-import CenteredLoading from '@/components/CenteredLoading.vue';
 import FeedbackAlert from '@/components/FeedbackAlert.vue';
-import { ref } from 'vue';
-import { useQuery } from '@tanstack/vue-query';
+import { checkEAD } from 'utils';
+
+const showDialog = ref(false);
+const selectedEnrollment = ref<Enrollment | undefined>();
```

Comment:
não precisa tipar explicitamente o undefined se o valor inicial dele é esse
```suggestion
const selectedEnrollment = ref<Enrollment>();
```